src/style.css

```
@@ -0,0 +1,56 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.search-big,
+.search-small {
+  margin-top: 20px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.input-big,
+.input-small {
+  background: url(images/Search.svg) 25px center/19px 19px no-repeat;
+  padding-left: 62px;
+  width: 100%;
+  height: 68px;
+  font-size: 16px;
+  font-weight: 300;
+  border: none;
+  color: #3d4e61;
+  font-family: "Avenir", sans-serif;
```

move it to html and set fallback fonts
```suggestion
  font-family: inherit;
```